<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopySkinWeightTool</class>
 <widget class="QMainWindow" name="CopySkinWeightTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>936</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>851</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>491</width>
         <height>71</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>471</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 127);
border-color: rgb(85, 170, 127);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>         Copy Skin Weight Tool</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>521</width>
          <height>781</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>511</width>
             <height>221</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>511</width>
             <height>221</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string>copySkinWeight</string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>493</width>
              <height>172</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QGroupBox" name="baseMesh_GBOX">
                 <property name="minimumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <italic>true</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="title">
                  <string>baseMesh</string>
                 </property>
                 <widget class="QWidget" name="layoutWidget">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>30</y>
                    <width>223</width>
                    <height>72</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLineEdit" name="baseMesh_LET">
                     <property name="minimumSize">
                      <size>
                       <width>221</width>
                       <height>33</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>221</width>
                       <height>33</height>
                      </size>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="baseMesh_BTN">
                     <property name="minimumSize">
                      <size>
                       <width>221</width>
                       <height>31</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>221</width>
                       <height>31</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>13</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(85, 170, 127);</string>
                     </property>
                     <property name="text">
                      <string>select</string>
                     </property>
                     <property name="default">
                      <bool>true</bool>
                     </property>
                     <property name="flat">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="targetMesh_GBOX">
                 <property name="minimumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <italic>true</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="title">
                  <string>targetMesh</string>
                 </property>
                 <widget class="QWidget" name="layoutWidget">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>30</y>
                    <width>223</width>
                    <height>72</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QLineEdit" name="targetMesh_LET">
                     <property name="minimumSize">
                      <size>
                       <width>221</width>
                       <height>33</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>221</width>
                       <height>33</height>
                      </size>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="targetMesh_BTN">
                     <property name="minimumSize">
                      <size>
                       <width>221</width>
                       <height>31</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>221</width>
                       <height>31</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>13</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(85, 170, 127);</string>
                     </property>
                     <property name="text">
                      <string>select</string>
                     </property>
                     <property name="default">
                      <bool>true</bool>
                     </property>
                     <property name="flat">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="copySkin_BTN">
               <property name="minimumSize">
                <size>
                 <width>491</width>
                 <height>51</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>491</width>
                 <height>51</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(85, 170, 127);</string>
               </property>
               <property name="text">
                <string>Copy Skin Weight To TARGET!</string>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="minimumSize">
            <size>
             <width>511</width>
             <height>221</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>511</width>
             <height>221</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string>Another copySkinWeight</string>
           </property>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>493</width>
              <height>172</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QGroupBox" name="baseMesh_GBOX_2">
                 <property name="minimumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <italic>true</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="title">
                  <string>baseName</string>
                 </property>
                 <widget class="QWidget" name="layoutWidget_4">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>30</y>
                    <width>223</width>
                    <height>72</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QLineEdit" name="anotherBase_LED">
                     <property name="font">
                      <font>
                       <pointsize>17</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="targetMesh_GBOX_2">
                 <property name="minimumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>241</width>
                   <height>111</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <italic>true</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="title">
                  <string>targetName</string>
                 </property>
                 <widget class="QWidget" name="layoutWidget_5">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>30</y>
                    <width>223</width>
                    <height>72</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QLineEdit" name="anotherTarget_LED">
                     <property name="font">
                      <font>
                       <pointsize>17</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="Another_copySkin_BTN">
               <property name="minimumSize">
                <size>
                 <width>491</width>
                 <height>51</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>491</width>
                 <height>51</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(85, 170, 127);</string>
               </property>
               <property name="text">
                <string>Another Copy Skin Weight</string>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>511</width>
             <height>151</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>511</width>
             <height>151</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="title">
            <string>Other Input ( not Skin Cluster )</string>
           </property>
           <widget class="QListWidget" name="otherInput_LWG">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>191</width>
              <height>121</height>
             </rect>
            </property>
           </widget>
           <widget class="QListWidget" name="blendShape_LWG">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>20</y>
              <width>191</width>
              <height>51</height>
             </rect>
            </property>
           </widget>
           <widget class="QListWidget" name="wrap_LWG">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>90</y>
              <width>191</width>
              <height>51</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>20</y>
              <width>91</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>BlendShape</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>90</y>
              <width>81</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>     wrap</string>
            </property>
           </widget>
           <zorder>otherInput_LWG</zorder>
           <zorder>blendShape_LWG</zorder>
           <zorder>wrap_LWG</zorder>
           <zorder>label_2</zorder>
           <zorder>label_5</zorder>
           <zorder>groupBox_4</zorder>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="title">
            <string>New added  Mesh</string>
           </property>
           <widget class="QListWidget" name="newAddedMesh_LWG">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>191</width>
              <height>121</height>
             </rect>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>20</y>
              <width>297</width>
              <height>35</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>91</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>91</width>
                 <height>31</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>BodyMesh :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="bodyMesh_LED">
               <property name="minimumSize">
                <size>
                 <width>141</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>141</width>
                 <height>33</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bodyMeshSel_BTN">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>51</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>SEL</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="copyToBody_BTN">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>60</y>
              <width>291</width>
              <height>81</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 170, 127);</string>
            </property>
            <property name="text">
             <string>COPY TO BODY</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>label</zorder>
       <zorder>groupBox</zorder>
       <zorder>verticalLayoutWidget</zorder>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
